### PR TITLE
get IdVerificationRequest, send IdVerification data

### PR DESCRIPTION
**Pull request by skynet67**

### CODE REVIEW

File: fitpay/src/main/java/com/fitpay/android/webview/models/IdVerification.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    private Boolean nfcCapable;
+
+    private IdVerification() {
+    }
+
+    /**
+     * send data to RTM
+     */
+    public void send() {
+        RxBus.getInstance().post(new RtmMessageResponse(this, RtmType.ID_VERIFICATION));
```
Comment (ssteveli) [inline fitpay/src/main/java/com/fitpay/android/webview/models/IdVerification.java]: Does this need the callback information?

Comment (skynet67) [inline fitpay/src/main/java/com/fitpay/android/webview/models/IdVerification.java]: according to the task https://fitpay.atlassian.net/browse/PLAT-3109 it doesn't need it

Comment (ssteveli) [inline fitpay/src/main/java/com/fitpay/android/webview/models/IdVerification.java]: Sorry, I meant at the RTM protocol level in the SDK itself in response to an RTM message from the webapp.  There is a `callbackId` value, I could have sworn that was returned back to the webapp.

Comment (skynet67) [inline fitpay/src/main/java/com/fitpay/android/webview/models/IdVerification.java]: I'll add it

**Comment by Polar-Beard** [comment]
I changed two variable types to Date instead of Integer, to make it more flexible for us.

**Pull request merged**